File: 0034-find-first-and-last-position-of-element-in-sorted-array/0034-find-first-and-last-position-of-element-in-sorted-array.py
class Solution:
    def searchRange(self, nums: List[int], target: int) -> List[int]:
        i=0
        j=len(nums)-1
        l=[]
        flag=False
        while(i<=j):
            if nums[i]==target:
                index1 = i
                while(i<j and nums[i]==nums[i+1]):
                    i += 1
                index2 = i
                l.append(index1)
                l.append(index2)
                flag=True
                break

            i += 1
            

        if flag:
            return l
        else:
            return [-1, -1]
        

File: 0034-find-first-and-last-position-of-element-in-sorted-array/0034-find-first-and-last-position-of-element-in-sorted-array.cpp
class Solution {
public:
    vector<int> searchRange(vector<int>& nums, int target) {
        vector<int> l;
        int index1=0;
        int index2=0;
        int i=0;
        int j=nums.size()-1;
        bool flag=false;
        if(nums.size()<2){
            if(nums.size()==0){
                return {-1,-1};
            }
            else if(nums[0]==target){
                return {0,0};
            }
        }
        else{
            while(i<=j){
                if(nums[i]==target){
                    index1 = i;
                    while(i<j && nums[i]==nums[i+1]){
                        
                        i++;
                    }
                    index2=i;
                    l.push_back(index1);
                    l.push_back(index2);
                    flag = true;
                    break;
                }
                else{
                    i++;
                }
            }
            
    }
    if(flag==true){
        return l;
    }
    else{
        return {-1,-1};
    }
        
    }
};